custom_components/tahoma/cover.py

```diff
@@ -8,6 +8,8 @@
 from pyoverkiz.enums import UIClass
 import voluptuous as vol
 
+from custom_components.tahoma import HomeAssistantOverkizData
```

Comment:
Should we just import from `.` here? (just like we do in core).

Reply:
I will mimic what we do on the Core side 👍